modules/local/mirtop_quant.nf

```diff
@@ -24,9 +24,12 @@ process MIRTOP_QUANT {
     script:
     def filter_species = params.mirgenedb ? params.mirgenedb_species : params.mirtrace_species
     """
-    mirtop gff --hairpin $hairpin --gtf $gtf -o mirtop --sps $filter_species ./bams/*
-    mirtop counts --hairpin $hairpin --gtf $gtf -o mirtop --sps $filter_species --add-extra --gff mirtop/mirtop.gff
-    mirtop export --format isomir --hairpin $hairpin --gtf $gtf --sps $filter_species -o mirtop mirtop/mirtop.gff
+    #Cleanup the GTF if mirbase html form is broken
+    GTF="$gtf"
+    sed 's/&gt;/>/g' \$GTF | sed 's#<br>#\\n#g' | sed 's#</p>##g' | sed 's#<p>##g' | sed -e :a -e '/^\\n*\$/{\$d;N;};/\\n\$/ba' > \${GTF}_html_cleaned.fa
+    mirtop gff --hairpin $hairpin --gtf \${GTF}_html_cleaned.fa -o mirtop --sps $filter_species ./bams/*
```

Comment:
Should this be gff / gtf instead of .fa? `\${GTF}_html_cleaned.fa`

Reply:
Good spot!

Reply:
It didnt affect the output though, but better to have it clean/cleared.